catalog/content/core/content-core-checksumplugin/src/main/java/ddf/content/plugin/checksum/Checksum.java

```diff
@@ -42,40 +42,37 @@ public Checksum(ChecksumProvider checksumProvider) {
 
     @Override
     public CreateRequest process(CreateRequest input) throws PluginExecutionException {
-        InputStream inputStream;
-
         if (input == null) {
             throw new IllegalArgumentException("CreateRequest input cannot be null");
         }
 
-        try {
-            inputStream = input.getContentItem().getInputStream();
-        } catch (IOException e) {
-            throw new PluginExecutionException("Unable to retrieve input stream for content item",
-                    e);
-        }
+        try (InputStream inputStream = input.getContentItem()
```

Comment:
@stustison nice. I was thinking try with resources was a good fix for this one. I'm wondering if we should use that more often rather than try/catch/finally.